src/app/personalData/personalData.module.scss

```diff
@@ -0,0 +1,220 @@
+.personalDataWrap {
+	font-size: clamp(20px, 1.6625vw, 42px);
+}
+
+.personalDataTopContainer {
+	position: relative;
+	background: url("../../../public/personalDataBackground.jpg");
+	color: var(--floral-white);
+	padding: 80px 80px 60px;
+
+	&__title {
+		line-height: 72%;
+		font-size: clamp(64px, 6.25vw, 160px);
+		font-weight: 700;
+		text-align: center;
+		margin: 241px 0 271px;
+	}
+}
+
+.bannerWrapper {
+	font-size: 14px;
+}
+
+.personalDataTopicsWrapper {
+	width: 93.75vw;
+	margin: 0 auto;
+}
+
+.personalDataTopicsContainer {
+	width: 83.16%;
```

Comment:
width: 83.2%;
try to avoid hundredths of a number in css rules except for those places where it is impossible to do without it